### DISCUSSION TITLE
Does this actually read barcodes?

### DISCUSSION THREAD

**Discussion by mana-dmg** [polarikus/flipper-zero_bc_scanner_emulator#4] [Q&A] [answered]
Not seeing that I can actually read a barcode with this. It shows up under apps/usb/bc_scanner.fap, but whenever I try to use it, it just shows an empty dir

**Comment by polarikus** [comment, marked as answer]
Please read the project description. This is a barcode scanner emulator, it does not actually read barcodes but only emulates the scanning process. 
You need to create a file with the extension .bc and add a string to it, which the application will send to the virtual COM port, thereby simulating the scanner.
